tests/typescript/conformance/classes/classDeclarations/classAbstractKeyword/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,889 @@
+    
+    static abstract foo_e();
+}~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+abstract class A {
+  abstract foo_a();
+
+  public abstract foo_b();
+  protected abstract foo_c();
+  private abstract foo_d();
+
+  public abstract foo_bb();
+  protected abstract foo_cc();
+  private abstract foo_dd();
+
+  static abstract foo_d();
+
+  static abstract foo_e();
+}
+
+`;
+
+exports[`classAbstractOverloads.ts 1`] = `
+abstract class A {
+    abstract foo();
+    abstract foo() : number;
+    abstract foo();
+    
+    abstract bar();
+    bar();
+    abstract bar();
+    
+    abstract baz();
+    baz();
+    abstract baz();
+    baz() {}
+    
+    qux();
+}
+
+abstract class B {
+    abstract foo() : number;
+    abstract foo();
+    x : number;
+    abstract foo();
+    abstract foo();
+}~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+abstract class A {
+  abstract foo();
+  abstract foo(): number;
+  abstract foo();
+
+  abstract bar();
+  bar();
+  abstract bar();
+
+  abstract baz();
+  baz();
+  abstract baz();
+  baz() {}
+
+  qux();
+}
+
+abstract class B {
+  abstract foo(): number;
+  abstract foo();
+  x: number;
+  abstract foo();
+  abstract foo();
+}
+
+`;
+
+exports[`classAbstractOverrideWithAbstract.ts 1`] = `
+class A {
+    foo() {}
+}
+
+abstract class B extends A {
+    abstract foo();
+}
+
+abstract class AA {
+    foo() {}
+    abstract bar();
+}
+
+abstract class BB extends AA {
+    abstract foo();
+    bar () {}
+}
+
+class CC extends BB {}
+
+class DD extends BB {
+    foo() {}
+}~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+class A {
+  foo() {}
+}
+
+abstract class B extends A {
+  abstract foo();
+}
+
+abstract class AA {
+  foo() {}
+  abstract bar();
+}
+
+abstract class BB extends AA {
+  abstract foo();
+  bar() {}
+}
+
+class CC extends BB {}
+
+class DD extends BB {
+  foo() {}
+}
+
+`;
+
+exports[`classAbstractProperties.ts 1`] = `
+abstract class A {
+    abstract x : number;
+    public abstract y : number;
+    protected abstract z : number;
+    private abstract w : number;
+    
+    abstract m: () => void; 
+    
+    abstract foo_x() : number;
+    public abstract foo_y() : number;
+    protected abstract foo_z() : number;
+    private abstract foo_w() : number;
+}~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+abstract class A {
+  x: number;
+  public y: number;
+  protected z: number;
+  private w: number;
+
+  m: () => void;
+
+  abstract foo_x(): number;
+  public abstract foo_y(): number;
+  protected abstract foo_z(): number;
+  private abstract foo_w(): number;
+}
+
+`;
+
+exports[`classAbstractSingleLineDecl.ts 1`] = `
+abstract class A {}
+
+abstract
+class B {}
+
+abstract
+
+class C {}
+
+new A;
+new B;
+new C;~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+abstract class A {}
+
+abstract;
+class B {}
```

Comment:
Yeah, the newline makes that `abstract` disconnected from the `class B`, so this would not be semantically valid TypeScript. It is clearly still parsable syntactically.